### PR TITLE
replace has-text- helpers with tailwind equivalent 

### LINKED ISSUE #9186: replace `has-text-` helpers with tailwind equivalent
- link #9013

### PR DESCRIPTION
**Pull request by Jarsen136**
- [x] Refactoring
  - [x] Closes #9186

### CODE REVIEW

File: components/collection/drop/GenerativePreview.vue [hunk 1]
```diff
@@ -33,12 +33,14 @@
...
-        class="flex items-center has-text-link fixed-right"
+        class="flex items-center text-k-blue hover:text-k-blue-hover fixed-right"
         rel="nofollow noopener noreferrer"
         target="_blank"
         role="link">
         {{ $t('img') }}
-        <NeoIcon icon="arrow-up-right" class="ml-1 has-text-link" />
+        <NeoIcon
+          icon="arrow-up-right"
+          class="ml-1 text-k-blue hover:text-k-blue-hover" />
```
Comment (roiLeo) [inline components/collection/drop/GenerativePreview.vue:43]: hover on icon 😭

**Review by roiLeo — APPROVED** [review]
I think we can had config to style default nuxt-link color instead of copy/paste same class each time.

**Comment by Jarsen136** [comment]
> I think we can had config to style default nuxt-link color instead of copy/paste same class each time.

I agree. It should either be overridden in the global css style, or provide a new component wrapped `nuxt-link`.

File: components/collection/drop/GenerativePreview.vue (hunk 1 above)
Comment (Jarsen136) [reply components/collection/drop/GenerativePreview.vue]: Updated. It dose not need anymore because there is `<a />` tag with blue color outside

**Comment by prury** [comment]
image

still checking

**Comment by Jarsen136** [comment]
> image
> 
> still checking

✅ Fixed

**Review by preschian — APPROVED** [review]

**Comment by roiLeo** [comment]
@prury I'll merge it for now, feel free to open new issue if something wrong occurs

**Pull request merged**